about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Big 5 Conservation</title>
+  <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+  <script src="https://unpkg.com/scrollreveal@4.0.9/dist/scrollreveal.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="mainc">
+  <main class="main" id="main">
+    <header class="header" id="header">
+      <div class="socials" id="socials">
+        <ul class="social-media">
+          <li class="socialmedia">
+            <a href="#" class="social-link"><i class="ri-facebook-fill"></i></a>
+          </li>
+          <li class="socialmedia">
+          <a href="#"><i class="ri-twitter-fill"></i></i></a>
+          </li>
+          <li class="socialmedia">
+            <a href="#" class="social-link">Swahili</a>
+          </li>
+          <li class="socialmedia">
+            <a href="#" class="social-link">My page</a>
+          </li>
+          <li class="socialmedia">
+            <a href="#" class="social-link">Logout</a>
+          </li>
+        </ul>
+      </div>
+      <nav class="nav container" id="nav">
+      <!--<div class="menu-toggle-icon lg-screen-hidden place-items-center" id="menu-toggle-icon">
+        <i class="ri-menu-3-line menu-open-icon" id="openmenu"></i>
+        <i class="ri-close-fill menu-close-icon" id="closemenu"></i>
+      </div>-->
```

Comment:
- [ ] Please remove the commented pieces of code, this will ensure that you have a clean codebase and keep it easy to read. 🤓 🛠️